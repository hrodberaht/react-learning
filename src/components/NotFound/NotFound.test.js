import React from 'react';
import { shallow } from 'enzyme';

import NotFound from './NotFound';

describe('<NotFound />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<NotFound />);
  });
  it('should render component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
