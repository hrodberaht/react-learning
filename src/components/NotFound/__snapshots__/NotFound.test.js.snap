// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should render component 1`] = `
<div>
  404 NOT FOUND
</div>
`;
